modules/component-store/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/ngrx/platform#readme",
   "peerDependencies": {
-    "@angular/core": "^11.0.0",
+    "@angular/core": "^12.0.0",
```

Comment:
I'm not sure, but will this also "catch" the RC versions of Angular?
Or should this be `^12.0.0-rc.0` (which seems like it should be based on https://semver.npmjs.com/).

Reply:
Fixed!